pkgs/applications/networking/instant-messengers/ricochet-refresh/default.nix

```
@@ -0,0 +1,84 @@
+{ makeDesktopItem
+, lib
+, stdenv
+, fetchurl
+, fontconfig
+, freetype
+, libxcb
+, libxkbcommon
+, xorg
+}:
+
+assert stdenv.hostPlatform.system == "x86_64-linux";
+
+let
+  version = "3.0.10";
+  rpath = lib.makeLibraryPath [
+    fontconfig
+    freetype
+    libxcb
+    libxkbcommon
+    stdenv.cc.cc.lib
+    xorg.xcbutilimage
+    xorg.xcbutilkeysyms
+    xorg.xcbutilrenderutil
+    xorg.xcbutilwm
+    xorg.libX11
+    xorg.libXext
+  ];
+  desktopItem = makeDesktopItem {
+    name = "ricochet-refresh";
+    exec = "ricochet-refresh";
+    icon = "ricochet-refresh";
+    desktopName = "Ricochet Refresh";
+    genericName = "Ricochet Refresh";
+    categories = "Network;InstantMessaging;";
+  };
+
+in stdenv.mkDerivation {
+  name = "ricochet-refresh-${version}";
+
+  # nix by default strips binaries (to save space?) but there is
+  # an outstanding issue where this breaks if rpath is set to a longer value
+  # using patchelf prior
+  # see: https://github.com/NixOS/patchelf/issues/10
```

```suggestion
  # stripping breaks if rpath is set to a longer value using patchelf prior
  # see: https://github.com/NixOS/patchelf/issues/10
```
There is no reason you do not want to have an unstripped binary in a production system.